### PR TITLE
Removed unknown/unavailable resections from relevant survival models

### PR DESCRIPTION
**Pull request by sjspielman**
This PR revisits two of the survival notebooks, `immune` and `tp53_telomerase`, which construct models using `extent_of_tumor_resection`. The models included unknown resections (denoted "Unavailable" or "Not reported" in the data), but these categories are wholly uninterpretable so they should not be included.

This PR updates the specific survival models and associated results where this predictor was included. 

There is one particularly notable result change that I observed - the model with `quantiseq cell type + fractions CD274 + extent of tumor resection` now only converges for HGG, not MB.

### CODE REVIEW

File: analyses/survival-analysis/survival-analysis_immune.Rmd [hunk 1]
```diff
@@ -138,7 +137,7 @@ stranded <- read_rds(file.path(data_dir, "pbta-gene-expression-rsem-fpkm-collaps
 # Process gene expression files to obtain PDL1 (aka CD274)
 cd274_expression <- bind_rows(
   extract_cd274_expression(polya, "polya"),
-  stranded_cd274 <- extract_cd274_expression(stranded, "stranded")
+  extract_cd274_expression(stranded, "stranded")
```
Comment (sjspielman) [inline analyses/survival-analysis/survival-analysis_immune.Rmd:140]: I noticed this and couldn't help myself 😬

File: analyses/survival-analysis/survival-analysis_immune.Rmd [hunk 2]
```diff
@@ -164,7 +163,8 @@ We use a cox regression to explore influence of PDL-1 expression on survival, wh
 fit_save_model(combined_df, 
                "CD274 + library",
                file.path(results_dir, "cox_terms_pdl1_library.RDS"), 
-               model_type = "multivariate")
+               model_type = "multivariate") %>%
+  print()
```
Comment (sjspielman) [inline analyses/survival-analysis/survival-analysis_immune.Rmd:167]: chunk was being difficult printing when I previewed, so I forced it here

File: analyses/survival-analysis/survival-analysis_immune.Rmd (hunk 1 above)
Comment (jharenza) [reply analyses/survival-analysis/survival-analysis_immune.Rmd:140]: too funny

**Review by jharenza — APPROVED** [review]
looks good to me

**Pull request merged**